client/codegen_pinvoke/Main.cs

```diff
@@ -110,6 +112,10 @@ public void Setup(CppSharp.Driver driver)
             options.GenerateClassTemplates = false;
             options.GenerateFinalizers = false;
 
+#if UNITY_IOS
+            options.GenerateInternalImports = true;

```

Comment:
Does it actually need to be platform specific?
Also, add a comment here, its not clear what this does.

Reply:
Yes, it can only be used for the iOS DLL. It replaces the argument in all `DllImport`s with `__Internal` for static linking.